fs/fcb/include/fcb/fcb.h

```diff
@@ -139,17 +149,57 @@ int
 fcb_offset_last_n(struct fcb *fcb, uint8_t entries,
         struct fcb_entry *last_n_entry);
 
+/**
+ * Finds sector range for given fcb sector.
+ */
+struct sector_range *fcb_get_sector_range(const struct fcb *fcb, int sector);
+
+int fcb_offset_in_sector(const struct fcb *fcb, int offset, int sector);
+
+int fcb_get_sector_info(const struct fcb *fcb, int sector,
+    struct fcb_sector_info *info);
+
+int fcb_get_sector_loc(const struct fcb *fcb, int sector,
+    struct fcb_entry *entry);
+
+int fcb_get_sector_offset(const struct fcb *fcb, int sector);
+int fcb_entry_get_sector_offset(const struct fcb_entry *loc);
+
+int fcb_write_to_sector(struct fcb_entry *loc, int off,
+    const void *buf, int len);
+
+int fcb_read_from_sector(struct fcb_entry *loc, int off, void *buf, int len);
```

Comment:
It's tedious, but I think we should get in the habit of adding doxygen comments for all new public functions.  Otherwise, this stuff will probably never get documented.

Reply:
done